{"schemaVersion":{"major":0,"minor":3,"patch":0},"topicSections":[{"title":"Instance Properties","identifiers":["doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/failureReason","doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/helpAnchor","doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/recoverySuggestion"],"generated":true}],"metadata":{"modules":[{"name":"PostgresClientORM"}],"title":"LocalizedError Implementations","role":"collectionGroup"},"sections":[],"identifier":{"url":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/LocalizedError-Implementations","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM","doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError"]]},"kind":"article","variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/postgresclientorm\/tableobjecterror\/localizederror-implementations"]}],"references":{"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError":{"url":"\/documentation\/postgresclientorm\/tableobjecterror","role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"TableObjectError"}],"abstract":[],"title":"TableObjectError","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError","type":"topic","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"TableObjectError"}]},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/helpAnchor":{"url":"\/documentation\/postgresclientorm\/tableobjecterror\/helpanchor","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"helpAnchor"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"text":"?","kind":"text"}],"role":"symbol","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/helpAnchor","title":"helpAnchor","kind":"symbol","type":"topic","abstract":[]},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/recoverySuggestion":{"url":"\/documentation\/postgresclientorm\/tableobjecterror\/recoverysuggestion","role":"symbol","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"text":"recoverySuggestion","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"abstract":[],"identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/recoverySuggestion","title":"recoverySuggestion","type":"topic","kind":"symbol"},"doc://PostgresClientORM/documentation/PostgresClientORM":{"type":"topic","url":"\/documentation\/postgresclientorm","title":"PostgresClientORM","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM","kind":"symbol","abstract":[],"role":"collection"},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/failureReason":{"url":"\/documentation\/postgresclientorm\/tableobjecterror\/failurereason","title":"failureReason","type":"topic","role":"symbol","kind":"symbol","abstract":[],"identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/failureReason","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"failureReason"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}]}}}